views/home.hbs

```diff
@@ -0,0 +1,67 @@
+{{> header}}
+
+
+
+<section class="section-offers">
```

Comment:
Invalid BEM, create BEM for Carousel. E.g.
.carousel > (ul.carousel__items > li.carousel__item > img.carousel__image) + div.carousel__navigation.carousel__navigation--prev + div.carousel__navigation.carousel__navigation--next + ul.carousel_pagination > li.carousel__pagination__dot

This will make it reusable across different pages.